pycaching/cache.py

```diff
@@ -624,6 +695,16 @@ def load(self):
 
         self.location = Point.from_string(root.find(id="uxLatLon").text)
 
+        self.lat = str(root.find(id="ctl00_ContentBody_Location").find("a")).split("=")[2].split("&")[0]
+
+        self.lon = str(root.find(id="ctl00_ContentBody_Location").find("a")).split("=")[3].split("&")[0]
+
+        self.id = str(root.find(id="ctl00_ContentBody_GeoNav_logButton")).split("=")[3].split("&")[0]
```

Comment:
This is definitely not a good idea to parse an URL. Please take a look at [urllib.parse](https://docs.python.org/3/library/urllib.parse.html) to know how do this correctly. 😉

On the other hand – I don't see any reason, why to parse this cache ID. We already have 2 different IDs (waypoint + guid) which are used elsewhere. So why should we want another ID, which wouldn't be used anywhere?